state/state.go

```diff
@@ -107,7 +107,21 @@ func readFromYaml(content []byte, file string) (*HelmState, error) {
 }
 
 func stringTemplate() *template.Template {
-	return template.New("stringTemplate").Funcs(sprig.TxtFuncMap())
+	funcMap := sprig.TxtFuncMap()
+	alterFuncMap(&funcMap)
+	return template.New("stringTemplate").Funcs(funcMap)
+}
+
+func alterFuncMap(funcMap *template.FuncMap) {
+	(*funcMap)["required_env"] = requiredEnv
```

Comment:
According the discussion made in this PR, I'd like to make it assigned to the key`"env"`

Reply:
sounds fine, but lets make sure we document it as a difference from sprig

Reply:
Good point! That's definitely helpful and valuable.